docs/README.md

```diff
@@ -72,43 +71,53 @@ title: Metabase documentation
 - [Models](./data-modeling/models.md)
 - [Metadata-editing](./data-modeling/metadata-editing.md)
 - [Field types](./data-modeling/field-types.md)
+- [Default formatting](./data-modeling/formatting.md)
```

Comment:
Hmm, this one is interesting. I'd think to look for it under config options (toggling how things look), rather than a modeling option (how the schema behaves, or how it's interpreted). 

Not a strong protest, just for thought :)